src/test/java/org/ga4gh/drs/utils/cache/AccessCacheTest.java

```diff
@@ -0,0 +1,5 @@
+package org.ga4gh.drs.utils.cache;
+
+public class AccessCacheTest {
+    
```

Comment:
Maybe add a TODO so we don't forget to write these

Reply:
thanks, I'm going to add tests for this class (forgot to add them before)